src/pubsub/mod_pubsub.erl

```diff
@@ -198,7 +199,9 @@
           max_subscriptions_node  = undefined,
           default_node_config     = [],
           nodetree                = <<"nodetree_", (?STDTREE)/binary>>,
-          plugins                 = [?STDNODE]
+          plugins                 = [?STDNODE],
+          backend                 = mnesia,
```

Comment:
actually for that case, I would use macro like `DEFAULT_BACKEND` in the beginning of the module